saleor/checkout/calculations.py

```diff
@@ -179,7 +235,7 @@ def fetch_checkout_prices_if_expired(
     address: Optional["Address"] = None,
     discounts: Optional[Iterable["DiscountInfo"]] = None,
     force_update: bool = False,
-) -> "Checkout":
+) -> Tuple["CheckoutInfo", Iterable["CheckoutLineInfo"]]:
     """Fetch checkout prices with taxes.
 
     Apply checkout prices with taxes from plugins and
```

Comment:
I wouldn't say explicitly from webhooks. First we try the "old" approach, where any tax lines are calculated separatly, then we call a "new" approach, where we call single method which will return all prices fully calculated.